news-analysis.py

```diff
@@ -40,449 +40,451 @@
-        if compiled_sentiment[coin] < NEGATIVE_SENTIMENT_THRESHOLD and headlines_analysed[coin] >= MINUMUM_ARTICLES and coins_in_hand[coin]>0:
-
-            # check the volume looks correct
-            print(f'preparing to sell {coins_in_hand[coin]} {coin} at {CURRENT_PRICE[coin+PAIRING]}')
-
-            amount_to_sell = calculate_one_volume_from_lot_size(coin+PAIRING, coins_in_hand[coin]*99.5/100)
-
-            if (testnet):
-                # create test order before pushing an actual order
-                test_order = client.create_test_order(symbol=coin+PAIRING, side='SELL', type='MARKET', quantity=amount_to_sell)
+                        # print order confirmation to the console
+                        print(f"order {order[0]['orderId']} has been placed on {coin} with {order[0]['origQty']} coins sold for {sold_at} each at {utc_time}")
+                    else:
+                        print('Could not get last order from Binance!')
 
-            # try to create a real order if the test orders did not raise an exception
-            try:
-                buy_limit = client.create_order(
-                    symbol=coin+PAIRING,
-                    side='SELL',
-                    type='MARKET',
-                    quantity=amount_to_sell
-                )
-
-            #error handling here in case position cannot be placed
-            except Exception as e:
-                print(e)
-
-            # run the else block if the position has been placed and return some info
             else:
-                # set coin to 0
-                coins_in_hand[coin]=0
-                # retrieve the last order
-                order = client.get_all_orders(symbol=coin+PAIRING, limit=1)
-
-                if order:
-                    # convert order timsestamp into UTC format
-                    time = order[0]['time'] / 1000
-                    utc_time = datetime.fromtimestamp(time)
-
-                    # grab the price of CRYPTO the order was placed at for reporting
-                    sold_at = CURRENT_PRICE[coin+PAIRING]
-
-                    # print order condirmation to the console
-                    print(f"order {order[0]['orderId']} has been placed on {coin} with {order[0]['origQty']} coins sold for {sold_at} each at {utc_time}")
-                else:
-                    print('Could not get last order from Binance!')
-
-        else:
-            print(f'Sentiment not negative enough for {coin}, not enough headlines analysed or not enough {coin} to sell: {compiled_sentiment[coin]}, {headlines_analysed[coin]}')
+                print(f'Sentiment not negative enough for {coin}, not enough headlines analysed or not enough {coin} to sell: {compiled_sentiment[coin]}, {headlines_analysed[coin]}')
 
 
 if __name__ == '__main__':
+    argument_parser = ArgumentParser('Binance news sentiment bot.', formatter_class=ArgumentDefaultsHelpFormatter)
```

Comment:
Could you add docker commands and arguments in the README file? I would like to try this on local.

Reply:
@rushabhshah011 You are welcome)